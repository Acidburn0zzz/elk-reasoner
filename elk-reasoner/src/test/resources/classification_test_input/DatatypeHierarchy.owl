Prefix(xsd:=<http://www.w3.org/2001/XMLSchema#>)
Prefix(owl:=<http://www.w3.org/2002/07/owl#>)
Prefix(:=<http://www.semanticweb.org/Test.owl#>)
Prefix(xml:=<http://www.w3.org/XML/1998/namespace>)
Prefix(rdf:=<http://www.w3.org/1999/02/22-rdf-syntax-ns#>)
Prefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)

Ontology(<http://www.semanticweb.org/Test.owl>

EquivalentClasses(:AA DataSomeValuesFrom(:prop rdfs:Literal))
EquivalentClasses(:AB DataSomeValuesFrom(:prop xsd:dateTime))
EquivalentClasses(:AC DataSomeValuesFrom(:prop xsd:dateTimeStamp))
EquivalentClasses(:AD DataSomeValuesFrom(:prop xsd:base64Binary))
EquivalentClasses(:AE DataSomeValuesFrom(:prop xsd:hexBinary))
EquivalentClasses(:AF DataSomeValuesFrom(:prop xsd:anyURI))
EquivalentClasses(:AG DataSomeValuesFrom(:prop owl:real))
EquivalentClasses(:AH DataSomeValuesFrom(:prop owl:rational))
EquivalentClasses(:AI DataSomeValuesFrom(:prop xsd:decimal))
EquivalentClasses(:AJ DataSomeValuesFrom(:prop xsd:integer))
EquivalentClasses(:AK DataSomeValuesFrom(:prop xsd:nonNegativeInteger))
EquivalentClasses(:AL DataSomeValuesFrom(:prop rdf:PlainLiteral))
EquivalentClasses(:AM DataSomeValuesFrom(:prop xsd:string))
EquivalentClasses(:AN DataSomeValuesFrom(:prop xsd:normalizedString))
EquivalentClasses(:AO DataSomeValuesFrom(:prop xsd:token))
EquivalentClasses(:AP DataSomeValuesFrom(:prop xsd:Name))
EquivalentClasses(:AQ DataSomeValuesFrom(:prop xsd:NMTOKEN))
EquivalentClasses(:AR DataSomeValuesFrom(:prop xsd:NCName))
EquivalentClasses(:AS DataSomeValuesFrom(:prop rdf:XMLLiteral))
)