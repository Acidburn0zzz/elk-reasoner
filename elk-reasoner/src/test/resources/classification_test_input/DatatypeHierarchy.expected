Prefix(xsd:=<http://www.w3.org/2001/XMLSchema#>)
Prefix(owl:=<http://www.w3.org/2002/07/owl#>)
Prefix(:=<http://www.semanticweb.org/Test.owl#>)
Prefix(xml:=<http://www.w3.org/XML/1998/namespace>)
Prefix(rdf:=<http://www.w3.org/1999/02/22-rdf-syntax-ns#>)
Prefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)

Ontology(<http://www.semanticweb.org/Test.owl>

SubClassOf(<http://www.semanticweb.org/Test.owl#AA> owl:Thing)
SubClassOf(<http://www.semanticweb.org/Test.owl#AB> <http://www.semanticweb.org/Test.owl#AA>)
SubClassOf(<http://www.semanticweb.org/Test.owl#AC> <http://www.semanticweb.org/Test.owl#AB>)
SubClassOf(<http://www.semanticweb.org/Test.owl#AD> <http://www.semanticweb.org/Test.owl#AA>)
SubClassOf(<http://www.semanticweb.org/Test.owl#AE> <http://www.semanticweb.org/Test.owl#AA>)
SubClassOf(<http://www.semanticweb.org/Test.owl#AF> <http://www.semanticweb.org/Test.owl#AA>)
SubClassOf(<http://www.semanticweb.org/Test.owl#AG> <http://www.semanticweb.org/Test.owl#AA>)
SubClassOf(<http://www.semanticweb.org/Test.owl#AH> <http://www.semanticweb.org/Test.owl#AG>)
SubClassOf(<http://www.semanticweb.org/Test.owl#AI> <http://www.semanticweb.org/Test.owl#AH>)
SubClassOf(<http://www.semanticweb.org/Test.owl#AJ> <http://www.semanticweb.org/Test.owl#AI>)
SubClassOf(<http://www.semanticweb.org/Test.owl#AK> <http://www.semanticweb.org/Test.owl#AJ>)
SubClassOf(<http://www.semanticweb.org/Test.owl#AL> <http://www.semanticweb.org/Test.owl#AA>)
SubClassOf(<http://www.semanticweb.org/Test.owl#AM> <http://www.semanticweb.org/Test.owl#AL>)
SubClassOf(<http://www.semanticweb.org/Test.owl#AN> <http://www.semanticweb.org/Test.owl#AM>)
SubClassOf(<http://www.semanticweb.org/Test.owl#AO> <http://www.semanticweb.org/Test.owl#AN>)
SubClassOf(<http://www.semanticweb.org/Test.owl#AP> <http://www.semanticweb.org/Test.owl#AO>)
SubClassOf(<http://www.semanticweb.org/Test.owl#AQ> <http://www.semanticweb.org/Test.owl#AO>)
SubClassOf(<http://www.semanticweb.org/Test.owl#AR> <http://www.semanticweb.org/Test.owl#AP>)
SubClassOf(<http://www.semanticweb.org/Test.owl#AS> <http://www.semanticweb.org/Test.owl#AA>)
)
